lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -129,41 +124,6 @@ def ensure_https
       end
     end
 
-    # If the embed is hosted on the Archive, it's Dewplayer and can be replaced
-    # with <audio> tag(s).
-    #
-    # Refer to https://archiveofourown.org/admin_posts/250.
-    def replace_dewplayer
-      return unless source == :ao3 && node_name == "embed" && source_url.downcase.include?("dewplayer")
```

Comment:
I think we should also remove `ao3` from `WHITELIST_REGEXES` -- we only allowed it for Dewplayer embeds, as far as I can tell.